doc/book/connectors/__python.rst

```diff
@@ -44,3 +44,214 @@ Also there are several community-driven Python connectors:
+        -   Yes
+        -   FIXME
+        -   FIXME
+        -   FIXME
+
+    *   -   `SQL support <https://www.tarantool.io/en/doc/latest/reference/reference_sql/>`__
+        -   Yes (tests/test_op_sql.py)
+        -   No (using call server-side functions)
+        -   No
+        -   Yes (tarantool/connection.py)
+
+    *   -   `Interactive transactions <https://www.tarantool.io/en/doc/latest/book/box/stream/>`__
+        -   No
+        -   No
+        -   No
+        -   No (`issue #163 <https://github.com/tarantool/tarantool-python/issues/163>`__)
+
+    *   -   `Varbinary support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `UUID support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `Decimal support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   Almost (v2 branch)
+        -   No
+        -   No
+        -   No
+
+    *   -   `EXT_ERROR support <https://www.tarantool.io/ru/doc/latest/dev_guide/internals/msgpack_extensions/#the-error-type>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `[Soon] <https://github.com/tarantool/tarantool/discussions/6244>`__ Datetime support
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `box.session.push() responses <https://www.tarantool.io/ru/doc/latest/reference/reference_lua/box_session/push/>`__
+        -   Yes (see push_subscribe option and docs/pushes.rst)
+        -   No
+        -   No
+        -   No
+
+    *   -   `Session settings <https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_space/_session_settings/>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `Graceful shutdown <https://github.com/tarantool/tarantool/issues/5924>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `IPROTO_ID (feature discovering) <https://github.com/tarantool/doc/issues/2419>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   Support `CRUD <https://github.com/tarantool/crud>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   Support retrying when appropriate
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   Implicit reconnecting strategy
+        -   autoreconnect
+        -   No
+        -   Yes? (reconnect_max_attempts, reconnect_delay)
+        -   Yes? (reconnect_max_attempts, reconnect_delay), checking of connection liveness
+
+    *   -   Round robin failover
+        -   No
+        -   No
+        -   No
+        -   Yes (see tarantool/mesh_connection.py)
+
+    *   -   `[Soon] <https://github.com/tarantool/tarantool/issues/6256>`__ Master discovering
+        -   No
+        -   No
+        -   No
+        -   Yes? (using call server-side function)
```

Comment:
AFAIR, the `cluster_discovery_function` argument is about renewing the list of URI for the connection pool and does not provide ability to differentiate a master from a replica. There is https://github.com/tarantool/tarantool-python/issues/196 about master discovering.

Reply:
In another [comment](https://github.com/tarantool/doc/pull/2482#discussion_r776313461), we decided to don't mix 'master discovery' in connectors and 'master discovery' in Tarantool.

Reply:
Yep, but this comment still relevant: we have no master discovery in tarantool-python, but the table states the opposite.

Reply:
(And, while we're here, 'connection pool' could be removed from the parentheses, because we already say 'Yes' for the 'Connection pool' row.)

Reply:
Removed 'connection pool' and 'master discovery' from parentheses.